testcafe/tests/helpers/data.js

```diff
@@ -7,17 +8,37 @@ export const FOLDER_DATE_TIME = `Folder_${getCurrentDateTime()}`
 export const ALBUM_DATE_TIME = `Album_${getCurrentDateTime()}`
 // data files & path
 const homedir = require('os').homedir
-export const DOWNLOAD_PATH = `${homedir()}/Downloads/`
-export const DOWNLOAD_ZIP_PATH = `${DOWNLOAD_PATH}files.zip`
+export const DOWNLOAD_PATH = `${homedir()}/Downloads`
+export const DOWNLOAD_FOLDER_PATH = `${DOWNLOAD_PATH}/files.zip`
+export const filesList = []
 
-export const DATA_PATH = '../../data/' //all files in testcafe/data
-//Drive
-export const FILE_PDF = '[cozy]QA_table_ APPS.pdf'
-export const FILE_XLSX = '[cozy]QA_table_ APPS.xlsx'
+//Path to Zip file
+export const DATA_ZIP_PATH = path.resolve(
+  './testcafe/data/26fichiers-extensions.zip'
+)
+//Path where the zip is extract & other data
+export const DATA_PATH = path.resolve('./testcafe/data')
+export const FILE_FROM_ZIP_PATH = path.join(DATA_PATH, 'TYPES DE FICHIERS')
+
+//Drive (All files from zip)
+export const FILE_PDF = 'Cozy Drive for Desktop.pdf'
+export const FILE_XLSX = 'tests.xlsx'
```

Comment:
But these aren't all the files in the zip, are they?

Reply:
No, there are other files in the .zip
most of the files are used in the viewer feature test which use all files from the .zip 
and loops on file extension, so we don't need to define the exact name for each files.

FILE_PDF and FILE_EXT are use in the files/folders sharing test in which those specifics files are uploaded. so I need their exact name to upload them. 
(I can probably do something to find one PDF files in all the unzip files and use it, but as this was done this way before using the zip so I didn't re-work this part)

Reply:
Mmh ok. The downside is that if the content of the zip changes, this part of the code needs to be updated too. It would almost be worth having these 2 outside of the zip, just to be sure.

Do you think that's a big risk? If not I can live with it, but maybe just making the comment more explicit would be a good.

Reply:
I added a new comment here about those files.

For the now I don't think it will be a problem as I'm the only one writing e2e tests.
I would prefer to keep the files in the zip in order to keep our git repo "smalĺ" , but I guess it will depend on how often we need to change the data, and how many people will work on E2E tests.